src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.h

```diff
@@ -5,8 +5,9 @@
 #include <architecture/messaging/messaging.h>
 #include <architecture/msgPayloadDef/AttRefMsgPayload.h>
 #include <architecture/msgPayloadDef/EphemerisMsgPayload.h>
+#include <architecture/msgPayloadDef/FlybyDiagnosisMsgPayload.h>
 #include <architecture/msgPayloadDef/NavTransMsgPayload.h>
-#include "flybyPointAlgorithm.h"
+#include <fswAlgorithms/attGuidance/flybyPoint/flybyPointAlgorithm.h>
```

Comment:
:small_red_triangle_down: This should be changed back to what it was. The angled brackets imply a system include. The algorithm file is not.